04-shapes/Shapes.java

```diff
@@ -0,0 +1,31 @@
+import java.until.*;
+
+ public class Shapes{
+   public static void main(String[] args){
+     //Square//
+     double Side  = 5.8;
+
+     //Triangle//
+     double TriB = 10.6;
+     double TriH = 4.5;
+     //double Half = 0.5;
+
+     //variables holding the values//
+     double Squ = square(Side);
+     double Tri = triangle(TriB*TriH*0.5);
+
+    //Print//
+     System.out.println(Squ);
+     System.out.println(Tri);
+   }
+     public static double square(double s){
+       double Ans = s * s;
+       return Ans;
+     }
+
+       public static double triangle(double B, double H){
+         double Ans2 = B * H;
+         return Ans2;
+       }
+
```

Comment:
what happened to methods that calculate volume for 3D shapes?

Reply:
Oh yea sorry this is the wrong code.

Reply:
I see that you added the 3D shape volumes, nice job! Personally, I'm a student and I try not to be picky, but many of my professors tend to be, watch out for how you name your variables in the future, as your programs become more complex remember to give variables more appropriate names instead of just letters or, ex: Ans1, Ans2, Ans3. Aside from that good job! ^^